Thrive.csproj

```diff
@@ -187,6 +187,7 @@
     <Compile Include="src\gui_common\PlayButton.cs" />
     <Compile Include="src\gui_common\RadialMenu.cs" />
     <Compile Include="src\gui_common\RadialPopup.cs" />
+    <Compile Include="src\gui_common\SlideableTextureRect.cs" />
```

Comment:
I think the word "slidable" should be used instead of "slideable" the former seems more common in our codebase to make these types of words.